merkle/smt/writer_test.go

```diff
@@ -161,6 +161,48 @@ func testWriterBigBatch(t testing.TB) {
 	}
 
 	w := NewWriter(treeID, hasher, 256, 8)
+	rootUpd := update(ctx, t, w, &testAccessor{}, upd)
+
+	// Calculated using Python code.
```

Comment:
Include a link to the Python code? If I wanted to change the key/value scheme, or add more iterations, or another test, I would simultaneously be happy and frustrated with this tantalizing comment.

Reply:
Deleted this line, and clarified the other one a bit.